StyleCop.Analyzers/StyleCop.Analyzers.Test/SpacingRules/SA1016UnitTests.cs

```diff
@@ -96,7 +96,7 @@ class ClassName2
 {
 }
 
-[/*comment*/System.Obsolete]
+[/*comment*/ System.Obsolete]
```

Comment:
:warning: **Behavior change:** This code fix now only removes whitespace starting at the token where the violation is reported up through the next non-whitespace token.